File: predict.py
import numpy as np
from deep_model import deep_model
def predict(X,y,parameters):
    """
    Takes X (the data), y (the labels), and parameters (from the dense_nn() function).
    Returns predictions p for the chosen data set X.
    """
    m=X.shape[1]
    n=len(parameters)//2
    p=np.zeros((1,m))
    probs,caches=deep_model(X, parameters)
    for i in range(0,probs.shape[1]):
        if probs[0,i]>0.5:
            p[0,i]=1
        else:
            p[0,i]=0
    print("Accuracy: "  + str(np.sum((p == y)/m)))   
    return p

File: predict.jl
include("deep_model.jl")
function predict(X, y, parameters)
  m = size(X)[2]
  n = div(length(parameters), 2)
  p = Int[]
  probs, caches = deep_model(X, parameters)
  for i in 1:size(probs)[2]
    if probs[i] > 0.25
      push!(p, 1)
    else
      push!(p, 0)
    end
  end
  print(string("Accuracy ", sum((p .== y)/m)))
  return p
end